src/Common/src/System/Net/SafeSocketHandle.Windows.cs

```diff
@@ -9,8 +9,13 @@
 
 namespace System.Net.Sockets
 {
-    internal partial class SafeCloseSocket :
-#if DEBUG
+#if SYSTEM_NET_SOCKETS_DLL
+    public
+#else
+    internal
+#endif
```

Comment:
This is ok, but you could also just leave all of the above out (not specifying a visibility).  The main partial file will specify the visibility, and that'll be used with this one not defining any visibility.